packages/ppx_matches/ppx_matches.0.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+synopsis: "Small ppx to help check if a value matches a pattern"
+description: """
+Translates [%matches? <PATTERN>] into (function | <PATTERN> -> true | _ -> false)
+"""
+maintainer: "Will Robson <will at will robson dot uk>"
+authors: "Will Robson <will at will robson dot uk>"
+license: "MIT"
+homepage: "https://github.com/wrbx/ppx_matches"
+bug-reports: "https://github.com/wrbx/ppx_matches/issues"
+dev-repo: "git://github.com/wrbs/ppx_matches"
+depends: [ "ocaml" "ppxlib" "dune" ]
```

Comment:
```suggestion
depends: [
  "ocaml" {>= "4.04"}
  "ppxlib" {>= "0.15.0"}
  "dune" {>= "2.9"}
]
```
I'm not sure what are your minimal requirements for ppxlib and OCaml but this is most likely the bare minimum.
For dune, your own `dune-project` file says you require at least 2.9.